### PR TITLE
Swallow ListNamespacedJob http exceptions

### PR DESCRIPTION
**Pull request by sisuresh**
An attempt to deal with https://github.com/stellar/stellar-supercluster/issues/280. This will be rolled back if it doesn't work.

### CODE REVIEW

File: src/FSLibrary/StellarJobExec.fs [hunk 1]
```diff
@@ -179,6 +179,11 @@ type StellarFormation with
         : Map<string, bool> =
         let jst = new JobStatusTable()
         let mutable moreJobs = true
+
+        // We have seen sporadic http exceptions being thrown in the while loop below,
+        // so this is an attempt to see if we can just ignore the exceptions.
+        let mutable httpRequestErrorsAllowed = 5
```
Comment (anupsdf) [inline src/FSLibrary/StellarJobExec.fs:185]: How about naming this as numOfRetriesAllowed?

**Review by anupsdf — APPROVED** [review]
Looks good. Made a comment to call the variable as retry but its optional.

**Review by dmkozh — APPROVED** [review]

File: src/FSLibrary/StellarJobExec.fs (hunk 1 above)
Comment (sisuresh) [reply src/FSLibrary/StellarJobExec.fs]: I renamed to `numOfHttpRetriesAllowed` to make the retry point more clear.

**Comment by marta-lokhova** [comment]
I don't mind merging this as-is to test parallel catchup, though I think this is yet another instance pointing to a greater problem in client<>server interaction in SSC. It seems that the server occasionally drops the connection, and the client can't deal with it, causing missions to blow up. So we might see the same issue surface elsewhere. It might be worth looking into using retries system-wide, or keeping the connections alive (not sure if client supports this)

**Pull request merged**